fvm/UnknownQuantityHandler.cpp

```diff
@@ -131,6 +131,12 @@ real_t UnknownQuantityHandler::GetUnknownDataPreviousTime(const len_t qty) {
 real_t *UnknownQuantityHandler::GetUnknownData(const len_t qty) {
     return unknowns[qty]->GetData();
 }
+/**
+ * name: Name of unknown quantity.
+ */
+real_t *UnknownQuantityHandler::GetUnknownData(const std::string& name) {
+    return GetUnknownData(GetUnknownID(name));
```

Comment:
Hmm... Where is this method used? The ``GetUnknownID(...)`` method adds quite a bit of overhead so it should not be used in computationally intensive parts of the code.

Reply:
This replaced various occurences along the lines of

len_t id_x = unknows->GetUnknownId(...)
const real_t *data_x = unknowns->GetUnknownData(id_x)

where id_x wasn’t otherwise used. I don’t believe this was ever used in places with many calls — I’ll review this.

Reply:
> I don’t believe this was ever used in places with many calls — I’ll review this.

This was clearly not true. I fixed a few problematic uses in the latest commit, but will leave the ones you indicate below for the bounceaverage merge.